packages/server/src/graphql/middlewares/measurements.js

```diff
@@ -28,23 +27,17 @@ async function measurementValueValidation(root, args, _context, _info) {
 /**
  * @param {*} root
  * @param {object} args
- * @param {Schema.Context} context
+ * @param {Schema.Context} _context
  * @param {object} _info
  * @return {Promise<boolean>}
  */
-async function measurableStateValidation(root, args, context, _info) {
+async function measurementCompetitorTypeValidation(
```

Comment:
I'd also be curious to get a better name for this and other things here as well. Is this dealing with authorization maybe? If so, can be called something to do with authorization?

Reply:
No, it is not about "authorization" at all. I think it was an error to split that previous function on two function. Thanks. Will repair.

Reply:
No worries, that works

Reply:
On my first glance, I thought that the function has two logic purpose this is why I split this function on two. It was an error.